### PR TITLE
More structured UI

### PR DESCRIPTION
**Pull request by ghost**
Schema (keyspaces, column families) now easier to look at. Especially if it's growing large.

Schema
cassie1

Cluster
cassie2

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -188,8 +192,7 @@ select {
 }
 
 #cluster_info {
-	width: 500px;
-	padding-top: 10px;
+	width: 100% - 50px;
```
Comment (sebgiroux) [inline css/style.css]: don't use space between - and 50px :)

File: css/style.css [hunk 2]
```diff
@@ -233,7 +236,7 @@ a, a:visited {
 .composite_value b {color:blue;}
 
 #keyspaces {
-	width: 500px;
+	width: 100% - 50px;
```
Comment (sebgiroux) [inline css/style.css]: same here

File: views/cluster_info.php [hunk 3]
```diff
@@ -1,21 +1,32 @@
...
+		<td onclick="document.location.href='?nav=schema'"><a href="?nav=schema">Schema</a></td>
+		<td onclick="document.location.href='?nav=cluster'"><a href="?nav=cluster">Cluster</a></td>
+	</tr>
+</table>
+
+<?php echo $success_message; ?>
+<?php echo $error_message; ?>
+
+<?php
+if ($_GET['nav'] == "cluster") {
```
Comment (sebgiroux) [inline views/cluster_info.php]: use alternative syntax in views to be consistent with the rest, also single quotes for string and this should probably be a sub view

File: views/cluster_info.php [hunk 4]
```diff
@@ -34,17 +45,35 @@
...
 
+	<a href="keyspace_action.php?action=create" class="btn btn-large btn-primary" style="color: #fff; text-decoration: none;">Create New Keyspace</a>
+
+	<br /><br />
+
+	<table width="100%" class="table table-bordered table-striped kcfs">
 	<?php
 		$nb_ks = count($keyspaces_name);
 		for ($i = 0; $i < $nb_ks; $i++):
+			if ($i/2 == intval($i/2)) {
```
Comment (sebgiroux) [inline views/cluster_info.php]: alternative syntax here too

File: views/cluster_info.php [hunk 5]
```diff
@@ -34,17 +45,35 @@
...
 	<h3>Keyspaces and Column Families</h3>
 
+	<a href="keyspace_action.php?action=create" class="btn btn-large btn-primary" style="color: #fff; text-decoration: none;">Create New Keyspace</a>
+
+	<br /><br />
+
+	<table width="100%" class="table table-bordered table-striped kcfs">
 	<?php
 		$nb_ks = count($keyspaces_name);
 		for ($i = 0; $i < $nb_ks; $i++):
```
Comment (sebgiroux) [inline views/cluster_info.php:62]: It's probably missing logic after this for loop for when the last line is incomplete, the td should be colspan="2" and the tr should be closed

**Comment by sebgiroux** [comment]
I like it, I can merge after the code changes has been made :)

**Comment by ghost** [comment]
cool that you like it (because I like it, too ;) ). Made the changes to the files. :)

**Comment by sebgiroux** [comment]
You're very efficient :) Good work, thanks for this, keep them coming!

**Comment by sebgiroux** [comment]
Btw, I get a Notice: Undefined index: nav in \CassandraClusterAdmin\views\cluster_info.php on line 12

if ($_GET['nav'] == 'cluster'):

should be replaced with:

 if (isset($_GET['nav']) && $_GET['nav'] == 'cluster'):

Feel free to add notice error level to your PHP config :)

**Comment by ghost** [comment]
You're right. I just don't use that in my own applications because it doesn't stop them from working ;)

**Pull request merged**